Makefile.inc

```diff
@@ -19,4 +19,4 @@ HEADERS += $(wildcard $(USART)/code/headers/*.hpp)
 
 # search for queue.hpp
 SEARCH  += $(USART)/../datastructures/code/headers
-HEADERS += $(wildcard $(USART)/../datastructures/code/headers/queue.hpp)
\ No newline at end of file
+HEADERS += $(wildcard $(USART)/../datastructures/code/headers/queue.hpp)
```

Comment:
this is probably not necessary to include datastructures since it will be included by default from internal_communication but to be complete it is nice to have.

Reply:
I believe it does not pass travis without this

Reply:
it theory this does not influence travis since the inc is only used when including as a library but as said voor (volledigheid) is het wel netjes om te hebben.